setup.py

```diff
@@ -76,8 +77,15 @@ def install_system_packages(self):
                 manager, details = self.get_package_manager_details()
                 if manager:
                     if manager == "apt-get":
-                        subprocess.check_call(['sudo', 'apt-get', 'update'])
-                        subprocess.check_call(['sudo', 'apt-get', 'install', '-y'] + packages)
+                        # Check if 'sudo' is available
+                        if shutil.which('sudo'):
+                            try:
+                                subprocess.check_call(['sudo', 'apt-get', 'update'])
+                                subprocess.check_call(['sudo', 'apt-get', 'install', '-y'] + packages)
+                            except subprocess.CalledProcessError:
+                                print("Sudo command failed, trying without sudo.")
+                                subprocess.check_call(['apt-get', 'update'])
+                                subprocess.check_call(['apt-get', 'install', '-y'] + packages)
```

Comment:
If both fails we should add a print statement for manual installation

Reply:
Hi @arjunsuresh , made changes in commit https://github.com/mlcommons/cm4mlops/pull/504/commits/11d77ded973b5af4544cb6bb35b9a0ac82da7d44.